portal-ecs.yml

```diff
@@ -190,6 +190,9 @@ Parameters:
     Type: String
     Description: this key is used to generate random tokens for password reset and
       signup.
+  RailsSecretKeyBase:
+    Type: String
+    Description: Secret key base for cookie signing
```

Comment:
@dougmartin can you add guidance to the description about what the value should be? 
I'd guess a randomly generated string, longer is better. Are there any restrictions on characters. Or perhaps a better options for documentation purposes is to put a link to some Rails docs about it.